tests/proofs/simple-spec.k

```diff
@@ -0,0 +1,56 @@
+requires "kwasm-lemmas.md"
```

Comment:
For `#signed(ITYPE, 0)`, it probably can't tell that `0 <Int #pow1(ITYPE)`, even though `ITYPE` can only be `i32` or `i64`.

```
rule 0 <Int #pow1(_) => true [simplification]
```

and see if this works on Ana's branch.

Make sure to check the defniition of `#pow1` and make sure this is true.